### PR TITLE
Enabled RedisNamespace as cache store

### PR DESCRIPTION
**Pull request by araishikeiwai**

### CODE REVIEW

**Comment by grosser** [comment]
- versions should not be bumped in prs, please remove that
 - could you add a test ? (ok to add redis-namespace dev dependenct)

**Comment by grosser** [comment]
... I guess the tests just fail from the version bump ...

**Comment by araishikeiwai** [comment]
@grosser ah sorry i bumped the version so that my local machine can update the gem. will remove the version and added test

**Comment by araishikeiwai** [comment]
@grosser please check my spec

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.sw?
```
Comment (grosser) [inline .gitignore]: FYI if you add this to your global .gitignore you'll never have to add it to a project again

File: lib/cachy.rb [hunk 2]
```diff
@@ -156,7 +156,7 @@ def self.locales
   def self.wrap_cache(cache)
     if cache.respond_to? :read and cache.respond_to? :write
       cache
-    elsif cache.class.to_s == 'Redis'
+    elsif cache.class.to_s =~ /^Redis/
```
Comment (grosser) [inline lib/cachy.rb]: same as `.start_with?('Redis')`

**Comment by grosser** [comment]
nice tests!

**Comment by grosser** [comment]
0.4.2

**Pull request merged**